hub/core/meta/tensor_meta.py

```diff
@@ -29,12 +30,14 @@ class TensorMeta(Meta):
     max_shape: List[int]
     chunk_size: int
     length: int
+    compression: str
 
     @staticmethod
     def create(
         key: str,
         storage: StorageProvider,
         htype: str = DEFAULT_HTYPE,
+        compression: str = DEFAULT_COMPRESSION,
```

Comment:
you can remove this `compression` parameter here, as well as in the docstring. 

this parameter should only be in the `API`, the core shouldn't have this (notice `dtype` and `chunk_size` are missing in here, this is deliberate)